feature/UNI456-108-chatbot-page

justinriveracsus opened the pull request:
Chatbot page is set up, allowing communication between the user and the SacLifeBot. Adds ChatbotScreen.js and ChatbotStyles.js for use within the page. Dialogue boxes are styled to fit in with the dashboard screen UI. All aspects of the page (texts, boxes, icons) are set up to be changed individually. Slight modification to DashboardScreen.js to reorganize the nav bar.

nickLewisCSUS reviewed (commented):
Well Done! For the styles liked how you used the variables form the styles page we created. And the faded green in the chat and the faded gold as well. 

However, when tested on Android the enter button is slightly cutoff and I would add a more esthetically pleasing Title, instead of just SacLifeBot. (If you have time I think you can adjust these before we merge)

For the future we need to be able to maybe make separate chats instead of just one continuing one and you can select different chats with a menu button on the side of the screen.

justinriveracsus commented:
Make a couple tweaks. Currently do not have the Android emulator set up, so I used my best judgement as to what should be done.

1. Renamed the bot to HerkyBot, to match the bot already used by the school.
2. Made some UI tweaks to hopefully fix the send button being cut off.
3. Changed the value of the android input box keyboard lift from 20 to 80.

nickLewisCSUS commented:
Fixed the keyboard pop up for android and moved the send icon a little to the left.

justinriveracsus reviewed (commented):
Looks good. Thanks for tuning it up!
